package.json

```diff
@@ -62,7 +62,8 @@
         "webpack-merge": "^4.1.3"
     },
     "dependencies": {
-        "@babel/runtime": "^7.0.0-beta.51",
+        "@babel/polyfill": "^7.0.0-beta.56",
```

Comment:
I thought that @babel/polyfill wasn't need and @babel/preset-env work himself

Reply:
It is necessary